ambry-messageformat/src/main/java/com/github/ambry/messageformat/MessageFormatSend.java

```diff
@@ -241,6 +245,13 @@ private void fetchDataFromReadSet(MessageFormatMetrics metrics) throws MessageFo
     }
   }
 
+  public long getBlobCRCOffset(MessageReadSet messages, int idx) throws MessageFormatException, IOException {
```

Comment:
there is a better way to know the offset of CRC. When you call `store.get(storeKeys, storeGetOptions)`, it returns a `MessageReadSet` and a list of `MessageInfo`. In this case, we only have one `MessageInfo` and one `MessageReadSet`. The `MessageInfo`'s `getSize()` would return the size of this entire message, what we are trying to get is the crc of this message. As long as we know the message size, we can just do `MessageInfo.getSize() - MessageFormatRecord.Crc_Size`. Something like that. 

So you don't really need to have a new method to get the crc offset.